### ISSUE TITLE
QR code scan button unlabeled 

### ISSUE THREAD

**Issue by asadotzler** [mozilla-mobile/firefox-ios#5016]
When swiping through the browser controls with VoiceOver enabled, the QR code scan button is simply read as "button". I guess it's missing a label or something. 

Tested on iOS 12.0.1 with the current release version of Firefox.

**Comment by SimonBasca** [comment]
Still reproducible on iPhone Xr iOS.

Please not that the "Cancel" button is also unlabeled.
![IMG_69740EAFD224-1](https://user-images.githubusercontent.com/32130829/58323434-a1386800-7e2c-11e9-9909-74a61f52d1c5.jpeg)

**Comment by garvankeeley** [comment]
Fixed by #5039
